modules/test/src/main/java/io/oasp/module/test/common/base/BaseTest.java

```diff
@@ -0,0 +1,33 @@
+package io.oasp.module.test.common.base;
+
+import org.assertj.core.api.Assertions;
+import org.junit.After;
+import org.junit.Before;
+
+/**
+ * TODO shuber This type ...
+ *
+ * @author shuber
+ * @since 2.2.0
+ */
+public class BaseTest extends Assertions {
```

Comment:
BaseTest should IMHO already contain the static variable ```initialized``` that is managed in ```setUp``` and passed as argument to ```doSetUp```. This will also avoid the redundancy and provide a feature missing in junit4 (comes with junit5 BTW).